pystiche_papers/ulyanov_et_al_2016/_data.py

```diff
@@ -53,6 +53,10 @@ def content_transform(
     transforms_: List[transforms.Transform] = []
     if impl_params:
         if instance_norm:
+            # Necessary transformation, otherwise ValidRandomCrop will not work with too
+            # small images. This is therefore not in the reference implementation.
+            # TODO: ??
```

Comment:
How did they dealt with this?

Reply:
They simply take another image until the size fits.

Reply:
In that case, we should do that too. I'll look into it how to do this.